.gitlab-ci.yml

```diff
@@ -2,7 +2,7 @@ variables:
   # These are gitlab variables so that it's easier to do a manual deploy
   # If these are set witih value and description, then it gives you UI elements
   DOWNSTREAM_BRANCH:
-    value: "main"
```

Comment:
You can override these values when launching manual jobs within the gitlab pipeline (so you don't need to change this).

Reply:
I can approve once you merge the downstream :+1: (as this will require a commit on your end to put main as default)

Reply:
@r1viollet downstream merged, and main set back as default